sc/social/like/browser/templates/metadata.pt

```diff
@@ -1,18 +1,18 @@
-<tal:metas i18n:domain="sc.social.like" tal:condition="view/enabled">
-  <meta name="twitter:card" content="summary_large_image" />
-  <meta name="twitter:site" tal:attributes="content string:@${view/via}" />
-  <meta property="og:site_name" tal:attributes="content view/portal_title" />
-  <meta property="og:url" tal:attributes="content view/canonical_url" />
-  <meta property="og:type" tal:attributes="content view/type" />
-  <meta property="og:locale" tal:attributes="content view/language" />
-  <meta property="og:title" tal:attributes="content view/title" />
-  <meta property="og:description" tal:attributes="content view/description" />
-  <meta property="og:image" tal:attributes="content view/image_url" />
-  <tal:image tal:condition="nocall:view/image">
-    <meta property="og:image:height" tal:attributes="content view/image_height" />
-    <meta property="og:image:width" tal:attributes="content view/image_width" />
-    <meta property="og:image:type" tal:attributes="content view/image_type" />
-  </tal:image>
-  <meta tal:condition="view/admins" property="fb:admins" tal:attributes="content view/admins" />
-  <meta tal:condition="view/app_id" property="fb:app_id" tal:attributes="content view/app_id" />
-</tal:metas>
+<meta property="og:site_name" tal:attributes="content view/site_name" />
+<meta property="og:url" tal:attributes="content view/canonical_url" />
+<meta property="og:type" tal:attributes="content view/type" />
+<meta property="og:locale" tal:attributes="content view/language" />
+<meta property="og:title" tal:attributes="content view/title" />
+<meta property="og:description" tal:attributes="content view/description" />
+<meta property="og:image" tal:attributes="content view/image_url" />
+<tal:image tal:condition="nocall:view/image">
```

Comment:
no need to write **tal:condition** inside a **<tal:** tag, you can write just **condition** instead.